### PR TITLE
BA-1516-013: Respecting Religious Obligations

### PR DESCRIPTION
**Pull request by iangmhill**
**BA-1516-013** makes it general policy for all branches of the Student Government to avoid scheduling all-school events at the same time as important religious obligations.

### CODE REVIEW

**Comment by jamiesarahg** [comment]
Did we take out the clause in the funding? Also it says strongly recommends and not is required to. I wanted to note these differences if this is something we were ok with.

**Comment by iangmhill** [comment]
Are you ok with it? I removed the funding clause because then we'd be binding every single club and org on campus that was funded by the SAF. I'm not sure the SG should enforce scheduling restrictions on every club and org. We'd also have to make sure that everyone was trained to respect this rule (possible but logistically tricky). I'm imagining events like SLACfest - should we be able to tell SLAC when to plan events?

Regarding "recommending" vs "requiring," on further consideration, I think you're right. I'll remove the "whenever possible" qualification. If we're enacting this on the principle of respecting Olin's religious diversity, there shouldn't be exceptions to when that diversity is respected.

**Comment by jamiesarahg** [comment]
Great! Thank you Ian. When we wrote it in the funding clause, we didn’t want to impede clubs and orgs, only full school events, so I think as long as we take out the whenever possible it should be good.

Jamie

**Comment by edoyle1** [comment]
I concur with Jamie.

File: student_government_bylaws.md [hunk 1]
```diff
@@ -712,6 +712,46 @@ allocation as necessary.
...
+#### Section 4. Event Scheduling Guidelines.
+
+To respect the religious diversity of the student body, the Council and the four
+Student Government Committees do not schedule events directed at the entire
+student body on major religious obligations such that attending the event would
+impede students from participating in their religions' traditions. Events hosted
+by other organizations but funded by the Council are also prohibited from being
+scheduled on major religious obligations. While we hope that student groups
+funded by the Committee for Clubs and Organizations are considerate of their
+members' obligations, this restiction does not apply to clubs and organizations.
```
Comment (Oktober13) [inline student_government_bylaws.md]: Restriction is spelled wrong

File: student_government_bylaws.md [hunk 2]
```diff
@@ -712,6 +712,46 @@ allocation as necessary.
...
+| Easter        | All Day                                                                                                     | End of March - Mid April    |                    | Apr 16            | Apr 1              | Apr 21                | Apr 12             |
+| Rosh Hashana  | Evening services on night before and two days of services/family observance                                 | September - Early October   | Sundown Oct 2 -4   | Sundown,Sep 20-22 | Sundown Sep 9 - 11 | Sundown Sep 29- Oct 1 | Sundown Sep 18- 20 |
+| Yom Kippur    | Fasting from night before through next evening. All day in services                                         | Mid September - Mid October | Sundown Oct 11 -12 | Sundown Sep 29-30 | Sundown Sep 18-19  | Sundown Oct 8-9       | Sundown Sep 28-29  |
+| Passover      | religious dinners on first two nights, no levened bread or anything with wheat, corn or rice for eight days | End of March - April        | Sundown Apr 22-23  | Sundown Apr 10-11 | Sundown Mar 30-31  | Sundown Apr 19-20     | Sundown Apr 8-9    |
```
Comment (Oktober13) [inline student_government_bylaws.md]: Leavened bread is spelled incorrectly

**Comment by mathmitch7** [comment]
Amendment to the amendment: Remove the table, replace with a provision for the VPC to maintain a publicly-accessible table of days that will be recognized. This amendment passed.

**Comment by mathmitch7** [comment]
Amendment to the Amendment: Instead of prohibiting committees from scheduling events on these days, the organizations must review the schedule that is maintained before scheduling an event, and consider who they may be impacting before choosing an event. This amendment was approved.

**Pull request closed without merging**